src/core/services/api.ts

```diff
@@ -9,6 +9,10 @@ import {FileUploadResponse} from '../../api/types/modules';
 import {compose} from '../utils/lang';
 import * as random from '../utils/random';
 
+const CONSECUTIVE_RETRY_ATTEMPTS = 5;
+const RETRY_BACKOFF = 500;
```

Comment:
I assume the unit is milliseconds? Do we need a comment or are milliseconds the default? Maybe suffix the variable with `_DURATION`.

Reply:
Renamed to RETRY_DELAY_INCREMENT